package.json

```diff
@@ -8,8 +8,9 @@
     "test": "jest",
     "lint": "tslint -c tslint.json src/**/*.ts",
     "lint-fix": "tslint -c tslint.json src/**/*.ts --fix",
-    "build": "tsc -p ./tsconfig.node.json && cp src/ripemd.js dist && babel dist --out-dir dist",
```

Comment:
@jeffreyssmith2nd Now that we're not using babel, can we remove all of the babel-related dependencies now?